### PR TITLE
CV2-3776: paginate tipline messages

### PR DESCRIPTION
**Pull request by melsawy**
## Description

Paginate tipline message .

References: CV2-3776

## How has this been tested?

Implemented automated tests to query tipline messages using pagination .

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have added unit and feature tests, if the PR implements a new feature or otherwise would benefit from additional testing
- [ ] I have added regression tests, if the PR fixes a bug
- [ ] I have added logging, exception reporting, and custom tracing with any additional information required for debugging
- [ ] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [ ] My changes generate no new warnings
- [ ] If I added a third party module, I included a rationale for doing so and followed our current [guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/overview#Choose-the-%E2%80%9Cright%E2%80%9D-3rd-party-module)

### CODE REVIEW

**Review by caiosba — COMMENTED** [review]
Sawy, not clear to me here how we can get the previous and next items starting from a random message... I'll add more details to the ticket.

File: app/graph/types/team_type.rb [hunk 1]
```diff
@@ -301,6 +301,6 @@ def shared_teams
   end
 
   def tipline_messages(uid:)
-    object.tipline_messages.where(uid: uid).last(100)
```
Comment (DGaffney) [inline app/graph/types/team_type.rb:304]: do you still need a limit on this query?

Comment (melsawy) [reply app/graph/types/team_type.rb:304]: No and I removed the limit and sort items by id `DESC`

**Comment by melsawy** [comment]
@caiosba I added a unit test to get next items starting from a random message and updated the ticket with more details

**Review by caiosba — APPROVED** [review]

**Comment by caiosba** [comment]
> @caiosba I added a unit test to get next items starting from a random message and updated the ticket with more details

Thanks Sawy for addressing the review comments! Looks good now!

**Pull request merged**